TabulateSmarterTestContentPackage/Tabulator.cs

```diff
@@ -1255,6 +1228,18 @@ void CheckDependencyInManifest(ItemContext it, string dependencyFilename, string
             }
         }
 
+        string GetAslType(ItemContext it, XmlDocument xml, XmlDocument xmlMetadata)
+        {
+            bool aslFound = CheckForAttachment(it, xml, "ASL", "MP4");
+            if (!aslFound) ReportUnexpectedFiles(it, "ASL video", "^item_{0}_ASL", it.ItemId);
+
+            bool aslInMetadata = string.Equals(xmlMetadata.XpEvalE("metadata/sa:smarterAppMetadata/sa:AccessibilityTagsASLLanguage", sXmlNs), "Y", StringComparison.OrdinalIgnoreCase);
+            if (aslInMetadata && !aslFound) ReportError(it, ErrCat.Metadata, ErrSeverity.Degraded, "Item metadata specifies ASL but no ASL in item.");
+            if (!aslInMetadata && aslFound) ReportError(it, ErrCat.Metadata, ErrSeverity.Tolerable, "Item has ASL but not indicated in the metadata.");
+
+            return (aslFound && aslInMetadata) ? "MP4" : string.Empty;
+        }
```

Comment:
@bredd  Unless I'm missing it, it looks like you consolidated duplicate code into this method, but I don't see a change in the logic.  Based on the PRs description I would expect something to change.  Maybe I'm just missing it?